src/app.module.ts

```diff
@@ -13,7 +15,11 @@ const envFilePath: string = path.join(
 
 @Module({
   imports: [ConfigModule.forRoot({ envFilePath, isGlobal: true })],
-  controllers: [AppController],
+  controllers: [AppController, LoginController],
   providers: [AppService],
 })
-export class AppModule {}
+export class AppModule implements NestModule {
+  configure(consumer: MiddlewareConsumer) {
+    consumer.apply(ApiToken).forRoutes({ path: "*", method: RequestMethod.ALL });
```

Comment:
Looks like you are applying the middle for all paths, even for the login paths. I think you will have to apply the middleware for the paths in `AppController`. Please check.